apps/builder/src/features/analytics/components/StatsCards.tsx

```diff
@@ -10,6 +10,16 @@ import {
 import { Stats } from 'models'
 import React from 'react'
 
+const calculateCompletionRate = (
+  totalCompleted: number,
+  totalStarts: number
+): string => {
+  if (!totalCompleted || !totalStarts) return 'Not available'
+  if (isNaN(totalCompleted) || isNaN(totalStarts)) return 'Not available'
```

Comment:
These 2 lines can be removed because we know that `totalCompleted` and `totalStarts` are defined numbers

```suggestion

```

Reply:
done